code/modules/mob/living/silicon/emote.dm

```diff
@@ -0,0 +1,99 @@
+/datum/emote/silicon
+	mob_type_allowed_typecache = list(/mob/living/silicon)
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/sound/silicon
+	mob_type_allowed_typecache = list(/mob/living/silicon)
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/silicon/boop
+	key = "boop"
+	key_third_person = "boops"
+	message = "boops."
+
+/datum/emote/sound/silicon/buzz
+	key = "buzz"
+	key_third_person = "buzzes"
+	message = "buzzes."
+	message_param = "buzzes at %t."
+	sound = 'sound/machines/buzz-sigh.ogg'
+
+/datum/emote/sound/silicon/buzz2
+	key = "buzz2"
+	message = "buzzes twice."
+	sound = 'sound/machines/buzz-two.ogg'
+
+/datum/emote/sound/silicon/chime
+	key = "chime"
+	key_third_person = "chimes"
+	message = "chimes."
+	sound = 'sound/machines/chime.ogg'
+
+/datum/emote/sound/silicon/honk
+	key = "honk"
+	key_third_person = "honks"
+	message = "honks."
+	vary = TRUE
+	sound = 'sound/items/bikehorn.ogg'
```

Comment:
Shouldn't this only be available to clownborgs?

Reply:
check MODULE_IS_A_CLOWN here